shared/src/commonMain/kotlin/co/nimblehq/ic/kmm/suv/domain/model/QuestionAndAnswers.kt

```diff
@@ -0,0 +1,22 @@
+package co.nimblehq.ic.kmm.suv.domain.model
+
+data class QuestionAndAnswers(
+    val questionIndex: Int,
+    val answerInputs: List<AnswerInput>
+)
+
+sealed class AnswerInput {
+
+    abstract val id: String
+
+    data class Select(override val id: String) : AnswerInput()
+
+    data class Content(override val id: String, val content: String) : AnswerInput()
+}
```

Comment:
optional: another way to override the `id` field with a sealed class is passing it into the root class.

```suggestion
sealed class AnswerInput(
    open val id: String
) {

    data class Select(override val id: String) : AnswerInput(id)

    data class Content(override val id: String, val content: String) : AnswerInput(id)
}
```